src/main/java/org/gaul/s3proxy/nio2blob/AbstractNio2BlobStore.java

```diff
@@ -792,6 +792,9 @@ public final BlobAccess getBlobAccess(String container, String key) {
         Set<PosixFilePermission> permissions;
         try {
             permissions = Files.getPosixFilePermissions(path);
+        } catch (UnsupportedOperationException uoe) {
```

Comment:
This seems fine but please add similar clauses to the other uses of `getPosixFilePermissions`.